packages/integrations/image/src/loaders/index.ts

```diff
@@ -75,6 +102,44 @@ export interface TransformOptions {
 	 * @example "16:9" - strings can be used in the format of `{ratioWidth}:{ratioHeight}`.
 	 */
 	aspectRatio?: number | `${number}:${number}`;
+	/**
+	 * How the image should be resized to fit both `height` and `width`.
+	 *
+	 * @default 'cover'
+	 */
+	fit?: CropFit;
+	/**
+	 * The color used to fill the background when `fit` is set to `contain`.
+	 *
+	 * @default 'rgba(0, 0, 0, 1)'
+	 */
+	background?: string;
+	/**
+	 * Position of the crop when fit is `cover` or `contain`.
+	 *
+	 * @default 'centre'
+	 */
+	position?: CropPosition;
+	/**
+	 * The kernel to use for cropping.
+	 *
+	 * @default 'lanczos3'
+	 */
+	kernel?: CropKernel;
+	/**
+	 * Prevent enlarging if the image's width or height are already less than
+	 * the specified dimensions.
+	 *
+	 * @default false
+	 */
+	withoutEnlargement?: boolean;
+	/**
+	 * Prevent reducing if the image's width or height are already greater than
+	 * the specified dimensions.
+	 *
+	 * @default false
+	 */
+	withoutReduction?: boolean;
```

Comment:
I'd lean towards removing these options for now and just trust the width/height provided by users

I could see adding these in the future, but I'm a little worried how much extra noise the lower-level flags will adds to our docs and support threads 😅

Reply:
Definitely felt like too much when adding them :+1:

Only when with it because they were the only remaining options.

Reply:
Yeah it's probably not a bad idea to remove `kernel` for now as well 🤷

Both sharp and Squoosh support it so that's not a concern at least, but for the sake of our docs that may be a flag we add later if a specific user request comes in for that level of control